aws-cpp-sdk-core/source/http/curl/CurlHttpClient.cpp

```diff
@@ -408,7 +408,7 @@ std::shared_ptr<HttpResponse> CurlHttpClient::MakeRequest(HttpRequest& request,
 
         if (m_isUsingProxy)
         {
-            curl_easy_setopt(connectionHandle, CURLOPT_PROXY, m_proxyHost.c_str());
+            curl_easy_setopt(connectionHandle, CURLOPT_PROXY, (m_proxyScheme + "://" + m_proxyHost).c_str());

```

Comment:
Yeah, probably. Thanks for catching that up. I will do the proposed change in a bit and update the pull request accordingly.